NEWS.md

```diff
@@ -1,3 +1,15 @@
+# iheiddown 0.7.4
+
+## Package
+
+* Fixed #42 where the `chapter_pdf()` function displayed placeholder text and headers for other unpreviewed chapters when knitting. 
+* Closed #26 by adding an automatic package citations function to the thesis template.
+* Updated the README to signal that new projects must be named "00-ProjectTitle" to avoid issues when merging Rmd files together.
+
+## Thesis
+
+* Closed #32 by changing Harry Potter references to LOTR references. 
```

Comment:
Maybe highlight that these have been changed in both Thesis and Syllabus outputs?